flake.nix

```diff
@@ -159,7 +159,7 @@
 
       fibonacci-zig-wasm = final.stdenv.mkDerivation {
         pname = "fibonacci";
-        version = "0.2.0";
```

Comment:
Hm, do I need to update this in any other examples I touch too?

Reply:
For rust, we parse Cargo.toml and use that version.
For languages without a similar mechanism (like Zig) - yes

Reply:
Do I need to be updating the version in Cargo.toml every time I make a change to any of the Rust examples, then? Is this the tag that gets used for uploading to Drawbridge?

Reply:
Yes and Yes (will be)